src/getStoredState.js

```diff
@@ -55,14 +60,26 @@ export default function getStoredState (config, onComplete) {
     onComplete(err, restoredState)
   }
 
+  function mapKey (key) {
+    return createFragmentedKey(key, store ? store.getState() : undefined)
+  }
+
   function passWhitelistBlacklist (key) {
-    if (whitelist && whitelist.indexOf(key) === -1) return false
-    if (blacklist.indexOf(key) !== -1) return false
+    if (whitelist && whitelist.map(mapKey).indexOf(key) === -1) return false
+    if (blacklist.map(mapKey).indexOf(key) !== -1) return false
     return true
   }
 
-  function createStorageKey (key) {
-    return `${keyPrefix}${key}`
+  function defaultFragmentKeyToReducerKey (key) {
```

Comment:
`defaultFragmentKeyToReducerKey` and `defaultCreateFragmentedKey` are the same and also they are just `identity` function 

`const identity = x => x`